test/react-native/batch-integration.tsx

```diff
@@ -467,33 +570,35 @@ describe('React Native', () => {
 
       const rendered = rtl.render(<ReduxBugDemo />)
 
-      const assertValuesMatch = (rendered) => {
+      type RenderedType = typeof rendered
+
+      const assertValuesMatch = (rendered: RenderedType) => {
         const [, boolFromSelector] =
           rendered.getByTestId('boolFromSelector').children
         const [, boolFromStore] = rendered.getByTestId('boolFromStore').children
         expect(boolFromSelector).toBe(boolFromStore)
       }
 
-      const clickButton = (rendered, testID) => {
+      const clickButton = (rendered: RenderedType, testID: string) => {
         const button = rendered.getByTestId(testID)
         rtl.fireEvent.press(button)
       }
 
-      const clickAndRender = (rendered, testID) => {
+      const clickAndRender = (rendered: RenderedType) => {
         // Note: Normally we'd wrap this all in act(), but that automatically
         // wraps your code in batchedUpdates(). The point of this bug is that it
         // specifically occurs when you are _not_ batching updates!
         clickButton(rendered, 'setTimeout')
```

Comment:
:hand: Yeah, I think there's actually a bug in this test here. It should be `clickButton(rendered, testID)` - I apparently missed passing that through when I was writing this :)

Can you restore the arguments for this section and fix this line?

Reply:
Okay, no problem

Reply:
Okay, no problem